docs/docs/101/architecture.md

```diff
@@ -1,3 +1,3 @@
 # Virtlet architecture
 
-See the detailed architecture description in Virtlet [docs](../../dev/architecture/)
+See the detailed architecture description in Virtlet [docs](../dev/architecture/)
```

Comment:
@lukaszo can you verify that? probably 101 urls are working in other way but for sure this will not work on gh.

Reply:
101 are handled on docs.virtlet.cloud so it's working only there. https://docs.virtlet.cloud/101/architecture/#virtlet-architecture

Reply:
I think that one should be reverted, others are ok.